### PR TITLE
Adjust Atlantic Electrical Heater to new standards and fix issues

### PR DESCRIPTION
**Pull request by iMicknl**
As mentioned by @tillstaff in, there are some bugs in the Atlantic Electrical Heater implementation.

https://github.com/iMicknl/ha-tahoma/pull/408

### CODE REVIEW

File: custom_components/tahoma/climate_devices/atlantic_electrical_heater.py [hunk 1]
```diff
@@ -52,35 +52,31 @@ def supported_features(self) -> int:
...
     @property
     def hvac_modes(self) -> List[str]:
         """Return the list of available hvac operation modes."""
-        return [*MAP_HVAC_MODES]
+        return [*HVAC_MODES_TO_TAHOMA]
+
+    async def async_set_hvac_mode(self, hvac_mode: str) -> None:
+        """Set new target hvac mode."""
+        await self.async_execute_command(
+            COMMAND_SET_HEATING_LEVEL, HVAC_MODES_TO_TAHOMA[hvac_mode]
```
Comment (tillstaff) [inline custom_components/tahoma/climate_devices/atlantic_electrical_heater.py:65]: This command won't work for the PASS_ACTUATOR device.

Here are the commands sent by the Cozytouch application : 

Set to Off (the command is named "off" without parameters): 
```
2021-06-01 10:22:50 INFO (MainThread) [custom_components.tahoma] 2021-06-01 10:21:12: off executed via Android 2.5.4#244 Set heating level in room c423fb05-92c5-49b4-a917-c7ac553e7ef4 on io://xxxx-xxxx-xxxx/2634956, with [].
```

Set to On (there is no command for "on", it's just set the preset to "comfort"): 
```
2021-06-01 10:22:50 INFO (MainThread) [custom_components.tahoma] 2021-06-01 10:21:37: setHeatingLevel executed via Android 2.5.4#244 Set heating level in room c423fb05-92c5-49b4-a917-c7ac553e7ef4 on io://xxxx-xxxx-xxxx/2634956, with ['comfort'].
```

File: custom_components/tahoma/climate_devices/atlantic_electrical_heater.py [hunk 2]
```diff
@@ -18,22 +18,22 @@
 
 COMMAND_SET_HEATING_LEVEL = "setHeatingLevel"
 
+CORE_ON_OFF_STATE = "core:OnOffState"
 IO_TARGET_HEATING_LEVEL_STATE = "io:TargetHeatingLevelState"
 
-PRESET_FREEZE = "Freeze"
-PRESET_FROST_PROTECTION = "frostprotection"
-PRESET_OFF = "off"
+PRESET_FREEZE = "freeze"
```
Comment (tillstaff) [inline custom_components/tahoma/climate_devices/atlantic_electrical_heater.py:24]: Why did you rename "frost protection" to "freeze"? Frostprotection seems well named but I could be wrong.
In french, the function is named "hors-gel" and it will just set the temperature of the heater to 7°C to avoid any freezing in the house.

Although, the following presets are needed too in the Atlantic_Electrical_Heater device : 
PRESET_COMFORT1 = "comfort-1"
PRESET_COMFORT2 = "comfort-2"

Comment (iMicknl) [reply custom_components/tahoma/climate_devices/atlantic_electrical_heater.py:24]: I didn't change it, in this version it wasn't fixed yet. I will fix it was part of this PR as well!

File: custom_components/tahoma/climate_devices/atlantic_electrical_heater.py (hunk 1 above)
Comment (iMicknl) [reply custom_components/tahoma/climate_devices/atlantic_electrical_heater.py:65]: Off should work already in this PR, but I understood that I have to rename `on` to `comfort`?

File: custom_components/tahoma/climate_devices/atlantic_electrical_heater.py (hunk 2 above)
Comment (tillstaff) [reply custom_components/tahoma/climate_devices/atlantic_electrical_heater.py:24]: Ok I see, I thought the commit was coming from the other PR, sorry.

So just the two presets are missing.

File: custom_components/tahoma/climate_devices/atlantic_electrical_heater.py (hunk 1 above)
Comment (tillstaff) [reply custom_components/tahoma/climate_devices/atlantic_electrical_heater.py:65]: I just test it and yes, it's working for "off" (it's not the same command sent by the cozytouch app but it doesn't matter).

It will work ok with "comfort", but you need to keep the "on" value somewhere for the "hvac_mode(self)" function.
